core/src/components/picker-column-internal/picker-column-internal.tsx

```diff
@@ -409,12 +414,12 @@ export class PickerColumnInternal implements ComponentInterface {
 
   get activeItem() {
     return getElementRoot(this.el).querySelector(
-      `.picker-item[data-value="${this.value}"]:not([disabled])`
+      `.picker-item[data-value="${this.value}"]:not(.picker-item-disabled)`
     ) as HTMLElement | null;
   }
 
   render() {
-    const { items, color, isActive, numericInput } = this;
+    const { items, color, disabled: pickerDisabled, isActive, numericInput } = this;
```

Comment:
Can you share why you are renaming the `disabled` variable here? I don't see any naming conflicts in the local scope of the `render` function.

Reply:
oh, i just did this to reduce confusion when it's used below, e.g. `const isItemDisabled = pickerDisabled || item.disabled || false;` seemed more clear to me than `const isItemDisabled = disabled || item.disabled || false;`